extensions/resource-deployment/src/test/services/apiService.test.ts

```diff
@@ -5,11 +5,15 @@
 
 import 'mocha';
 import assert = require('assert');
-import { apiService } from '../services/apiService';
+import { apiService } from '../../services/apiService';
 
-suite('API Service Tests', function (): void {
-	test('getAzurecoreApi returns azure api', () => {
+describe('API Service Tests', function (): void {
+	it('get azurecoreApi returns azure api', () => {
 		const api = apiService.azurecoreApi;
 		assert(api !== undefined);
 	});
+	it.skip('get azdataApi returns azdata api', () => {
```

Comment:
Why are you adding a skipped test?

Reply:
The test works locally on mac but fails for some reason on Linux. Should I remove it?

Reply:
I don't really see these tests as doing anything useful so I would say that's fine - we don't really need to test that the ext api export stuff is working correctly.